Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -40,27 +40,56 @@ public static class CoypuElementScopeExtensions
     /// where the <c>parentScope.Find*()</c> call could not be found.
     /// </summary>
     /// <param name="scope">The <see cref="ElementScope"/> which is asserted to exist.</param>
+    /// <exception cref="WebTestException">If the control cannot be found.</exception>

```

Comment:
We should probably merge this into a single Exception doc

Reply:
We found docs using this style.